### PR TITLE
ENH add elementary scoring function for quantiles, multi_metric, updated docu

### PR DESCRIPTION
**Pull request by mayer79**

### CODE REVIEW

**Review by lorentzenchr — CHANGES_REQUESTED** [review]
Rename elementary_score_expectile.R to elementary_scores.R or similar.

File: DESCRIPTION [hunk 1]
```diff
@@ -2,16 +2,16 @@ Type: Package
 Package: MetricsWeighted
 Title: Weighted Metrics, Scoring Functions and Performance
     Measures for Machine Learning
-Version: 0.4.0
-Date: 2019-12-02
+Version: 0.5.0
+Date: 2019-12-26
```
Comment (lorentzenchr) [inline DESCRIPTION:6]: This puts me under pressure ;-)

**Comment by lorentzenchr** [comment]
Otherwise LGTM 👍

**Comment by lorentzenchr** [comment]
This also closes #7.

File: DESCRIPTION (hunk 1 above)
Comment (mayer79) [reply DESCRIPTION:6]: Haha

**Pull request merged**